highlight/events-explainer.md

```diff
@@ -155,8 +245,7 @@ p1 pointerup
 
   6. Should we fire the pointer event on the Range that was hit, instead of the containing Highlight? It seems more developer friendly to create a single event listener on a Highlight than to create one listener per Range in the group.
   
-  7. Should the Highlight be the target of the same pointer event that is delivered to the parent Elements?
-  
+  7. Should the Highlight be the target of the same pointer event that is delivered to the parent Elements? This would enable Highlights to be PointerEvent targets, and could introduce the risk of breaking the expectations of current code as PointerEvents currently only have Nodes as their target.

```

Comment:
```suggestion
  7. Should the Highlight be the `target` of the pointer event that is propagated to the parent Elements? This would enable Highlights to be PointerEvent targets, and could introduce the risk of breaking the expectations of current code as PointerEvents currently only have Nodes as their target.

```